macosx/FileRenameSheetController.h

```diff
@@ -12,17 +12,6 @@
 @class Torrent;
 
 @interface FileRenameSheetController : NSWindowController
-{
```

Comment:
Can I recommend doing a separate PR that removes all of these 32-bit variables (in a bunch of different files)?

Reply:
Ok, I'll try.